task-list/app/views/people/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>People with Stuff to Do</h1>
+<% @people.each do |person| %>
+  <ul>
+    <li>
+    <%= link_to person.name.capitalize, "/people/#{person.id}" %>
+    <% all_tasks = person.tasks %>
+    <% incomplete_tasks = all_tasks.reject { |task| !task.completed_date.nil? }.length %>
```

Comment:
It seems like the use of this `reject` enumerable is something that should potentially be located somewhere else other than in your view